arviz/utils.py

```diff
@@ -664,3 +665,35 @@ def credible_interval_warning(credible_interval, hdi_prob):
 
     hdi_prob = credible_interval
     return hdi_prob
+
+
+def html_templates():
```

Comment:
I would change this python function to a class that of structure. Reason for this is that

1. Function doesn't do anything but return values
2. Defining objects in a function scope makes it hard to "get at them" without running function
3. In future use case we only want one of these things we still need to deal with the other 2 because with a function call we always get them
4. In the code where this is used it'll make it more clear that computation likely isn't happening but were just "grabbing a template"



```
HTMLTemplates:
    html_template = xyz
```

As a sidenote DataClasses would be great but they were introduced in 3.7 so if we use them here they will break ArviZ for Python 3.6 users

Reply:
Okay, I understand. I've changed it to class.